numba_cuda/numba/cuda/tests/cudapy/test_nested_calls.py

```diff
@@ -0,0 +1,55 @@
+"""
+Test problems in nested calls.
+Usually due to invalid type conversion between function boundaries.
```

Comment:
I imagine this is copy/pasted from Numba, and is no longer relevant? I think it would be fine just to remove it.

Reply:
Yes, I just copied it from numba and adapted for kernel run to be consistent